src/components/homepage/Homepage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { IReduxStore } from '../../types';
+
+const Homepage: React.FC<{}> = () => {
+    const props  = useSelector((state: IReduxStore) => state);
```

Comment:
The type of 'state' is RootState instead of IReduxStore
```suggestion
    const props  = useSelector((state: RootState) => state);
```